terraform/modules/backend/main.tf

```diff
@@ -0,0 +1,190 @@
+terraform {
+  required_version = "~>1.5.1"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~>5.0"
+    }
+  }
+}
+
+# The state replication bucket is kept in a separate region (eu-west-1) to the source bucket and where we generally
+# create our infrastructure (eu-west-2), so we define a provider here to be especially for this
+provider "aws" {
+  alias  = "ireland"
+  region = "eu-west-1"
+
+  assume_role {
+    role_arn = "arn:aws:iam::815624722760:role/developer"
+  }
+}
+
+#tfsec:ignore:aws-s3-block-public-acls:block_public_acls is set to true, this is a false flag
+#tfsec:ignore:aws-s3-block-public-policy:block_public_policy is set to true, this is a false flag
+#tfsec:ignore:aws-s3-enable-bucket-encryption:using AES256 encryption with key managed by Amazon S3 instead of KMS
+#tfsec:ignore:aws-s3-ignore-public-acls:ignore_public_acls is set to true, this is a false flag
+#tfsec:ignore:aws-s3-no-public-buckets:restrict_public_buckets is set to true, this is a false flag
+#tfsec:ignore:aws-s3-encryption-customer-key:using key managed by S3 because cloudposse backend module can't be configured with KMS
+#tfsec:ignore:aws-s3-enable-versioning:versioning not required for replica of a bucket with versioning enabled
+#tfsec:ignore:aws-s3-enable-bucket-logging:ignoring this warning temporarily while deciding if logging is necessary

```

Comment:
We may want to create a new bucket to contain server access logs for the replica bucket (currently we only log the main state bucket). We're only ignoring this warning for a few days so that we don't lose sight of this.